ci/yaml/okd-console.yaml

```diff
@@ -83,10 +83,12 @@ spec:
       serviceAccountName: console
       containers:
       - name: console
-        image: quay.io/openshift/origin-console:4.15
+        image: quay.io/openshift/origin-console:4.18
         env:
         - name: BRIDGE_USER_AUTH
           value: disabled
+        - name: BRIDGE_K8S_AUTH_BEARER_TOKEN
```

Comment:
@virteng hi, console v4.16 and above require some authentication method, for example k8s token, like here.